pytest_selenium/drivers/edge.py

```diff
@@ -1,14 +1,20 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from distutils.version import LooseVersion
+from selenium import __version__ as SELENIUM_VERSION
 
 
-def driver_kwargs(capabilities, driver_log, driver_path, **kwargs):
-    kwargs = {}
+def driver_kwargs(capabilities, driver_path, service_log_path, **kwargs):
```

Comment:
I'd like to keep the keyword argument as `driver_log` as this is already our abstraction from `log_path` and `service_log_path`.